src/app/public/modules/tabs/tabset.component.spec.ts

```diff
@@ -952,6 +956,24 @@ describe('Tabset component', () => {
       validateTabSelected(el, 1);
     }));
 
+    it('should handle two-way binding on `active` input', fakeAsync(() => {
```

Comment:
The original issue reported an infinite loop - should we test for that? We could spy on `activeChange` emitter and check to see if its only called a certain # of times.

Reply:
Good idea. Done.